cloudbreak/user/user.go

```diff
@@ -1,42 +1,115 @@
 package user
 
 import (
-	"github.com/hortonworks/cb-cli/cloudbreak/oauth"
 	"time"
 
+	"github.com/go-openapi/strfmt"
+	"github.com/go-openapi/swag"
+	"github.com/hortonworks/cb-cli/cloudbreak/oauth"
+	"github.com/hortonworks/cb-cli/dataplane/api/client/oidc"
+	"github.com/hortonworks/cb-cli/dataplane/api/client/roles"
+	"github.com/hortonworks/cb-cli/dataplane/api/client/users"
+
 	log "github.com/Sirupsen/logrus"
-	"github.com/hortonworks/cb-cli/cloudbreak/api/client/v1users"
-	"github.com/hortonworks/cb-cli/cloudbreak/api/model"
 	fl "github.com/hortonworks/cb-cli/cloudbreak/flags"
+	"github.com/hortonworks/cb-cli/dataplane/api/model"
 	"github.com/hortonworks/dp-cli-common/utils"
 	"github.com/urfave/cli"
 )
 
-var userListHeader = []string{"User ID", "Name"}
+var userListHeader = []string{"ID", "tenantID", "PreferredUsername", "Name", "Email"}
+var userInfoHeader = []string{"ID", "TenantID", "PrefferedUserName", "Name", "Email"}
+var roleDetailsHeader = []string{"ID", "Name", "DisplayName", "Service"}
+
+type UsersInfoOut struct {
+	User *model.UserInfo
+}
+
+func (u *UsersInfoOut) DataAsStringArray() []string {
+	return []string{
+		strfmt.UUID.String(u.User.ID),
```

Comment:
I think `u.User.ID.String()` would give the same result. What's the point using a method outside the struct in this case?

Reply:
Yes its same. Have modified the code